src/components/app/ServiceWorkerManager.css

```diff
@@ -0,0 +1,48 @@
+.serviceworker-ready-notice-wrapper {
```

Comment:
I tried to follow closely the guidelines from https://design.firefox.com/photon/components/message-bars.html#type-specific-style